### PR TITLE
Add QR code to connected wallet pop-up

### PR DESCRIPTION
**Pull request by rafaelzimmermann**
This make it easier to read the connected wallet address from another
device by using a QRCode reader.

It is common to have an exchange wallet on your phone and the station on your desktop. This will make it easier to read the station wallet connected address into my phone while making a deposit.

image

image

### CODE REVIEW

**Review by dricazenck — APPROVED** [review]

**Comment by simcheolhwan** [comment]
Thanks for this commit. 

This feature was already planned to be implemented. The Terra Station extension already has this function, so can you make it use the same module? Refer to the link below. Or I can do it myself.

https://github.com/terra-money/station-extension/blob/main/src/extension/components/WalletQR.tsx

**Comment by rafaelzimmermann** [comment]
Thanks! If you don't mind, I would like to do it 😄

**Comment by rafaelzimmermann** [comment]
@simcheolhwan I updated the PR

File: src/app/sections/Connected.module.scss [hunk 1]
```diff
@@ -8,3 +8,7 @@
   font-weight: var(--bold);
   margin-bottom: 4px;
 }
+
+button {
```
Comment (simcheolhwan) [inline src/app/sections/Connected.module.scss:12]: This will affect all buttons in the application.

**Review by simcheolhwan — CHANGES_REQUESTED** [review]

**Comment by simcheolhwan** [comment]
c58f3e5 
I couldn't explain everything, so I added some changes to your code.
This is what I wanted.

**Pull request merged**